book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] The `Book` and `Person` classes have a **_many-to-many_** relationship through the `Rental` class. This can be successfully implemented by using the `@rentals` array and the `add_rental` methods in both the `Book` and `Person` classes. This would enable their instances to keep records of all the rentals that belong to that instance. 👍 😎 

I would advise you to create the `add_rental` method in the `Book` class to successfully implement the relationship between both classes. The `add_rental` method should create a new instance of the `Rental` class. 💻 🖱️ 

#### Hint:

```ruby
def add_rental(date, person)
  Rental.new(date, self, person)
end
```